src/transformers/models/biogpt/modeling_biogpt.py

```diff
@@ -46,7 +46,6 @@
 _CONFIG_FOR_DOC = "BioGptConfig"
 
 
-# Copied from transformers.models.opt.modeling_opt.OPTLearnedPositionalEmbedding with OPT->BioGpt
```

Comment:
let's just add a TODO here or also update that model

Reply:
should I return the comment as well? it causes a fail when using make fixup.
I tried to look to update BioGPT, but it seems to be a lot of code from different models, so i didn't know if i should have touched it. I can work on it next.

Reply:
yes !